application/apis/narou/type.py

```diff
@@ -68,8 +68,74 @@ def is_valid_date_for_rank_type(rank_type, date):
 
 
 class OutType(Enum):
-    """なろうランキングout."""
+    """なろうAPI関連のout."""
 
     YAML = "yaml"
     JSON = "json"
     PHP = "php"
+
+
+class NarouOrderType(Enum):
+    """なろうAPIの出力順序."""
+
+    NEW = "new"
+    FAVORITE_NOVEL_COUNT = "favnovelcnt"
+    REVIEW_COUNT = "reviewcnt"
+    HYOKA = "hyoka"
+    HYOKA_ASC = "hyokaasc"
+    DAILY_POINT = "dailypoint"
+    WEEKLY_POINT = "weeklypoint"
+    MONTHLY_POINT = "monthlypoint"
+    QUARTER_POINT = "quarterpoint"
+    YEARL_YPOINT = "yearlypoint"
+    IMPRESSION_COUNT = "impressioncnt"
+    HYOKA_COUNT = "hyokacnt"
+    HYOKA_COUNT_ASC = "hyokacntasc"
+    WEEKLY = "weekly"
+    LENGTH_DESC = "lengthdesc"
+    LENGTH_ASC = "lengthasc"
+    GENERAL_FIRST_UP = "generalfirstup"
+    NCODED_DESC = "ncodedesc"
```

Comment:
```suggestion
    NCODE_DESC = "ncodedesc"
```
`NCODED_DESC` -> `NCODE_DESC` ですね 😓 
別コミットで改修するの面倒だと思うので `Commit suggestion` してもらうのが良いかも 👍 
![貼り付けた画像_2024_12_26_19_22](https://github.com/user-attachments/assets/2c8a41a8-c29b-44fa-836c-708ab6b35c67)

Reply:
@takuron1996 
↑ メンション忘れてました 🙏

Reply:
@watame 
[4b022bc](https://github.com/takuron1996/narou_novel_batch/pull/11/commits/4b022bcf35545c027933a50915e3680727a78c98)で対応しました。